run_e3sm.template.sh

```diff
@@ -0,0 +1,461 @@
+#!/bin/bash -fe
+
+# E3SM Water Cycle v2 run_e3sm script template.
+#
+# Inspired by v1 run_e3sm script as well as SCREAM group simplified run script.
+#
+# Bash coding style inspired by:
+# http://kfirlavi.herokuapp.com/blog/2012/11/14/defensive-bash-programming
+
+# TO DO:
+# - custom pelayout
+
+main() {
+
+# For debugging, uncomment libe below
```

Comment:
Noticed this typo here. Should I fix before merging?

Reply:
sure.

Reply:
Fixed.